components/org.wso2.carbon.identity.oauth.scope.endpoint/src/main/resources/api.identity.oauth2.scope.endpoint.yaml

```diff
@@ -35,8 +35,8 @@ host: localhost:9443
 # Will be prefixed to all paths.
 
 # For a tenant, basepath should be /t/{tenant_domain}/api/identity/oauth2/v1.0.
```

Comment:
If we are to provide the tenant path as the base path by default we can get rid of this comment.